src/test/java/uk/gov/hmcts/reform/sscs/factory/NotificationFactoryTest.java

```diff
@@ -104,11 +131,46 @@ public void setup() {
         when(notificationDateConverterUtil.toEmailDate(any(LocalDate.class))).thenReturn("1 January 2018");
     }
 
+    @Test
+    @Parameters({"APPELLANT, appellantEmail", "REPRESENTATIVE, repsEmail"})
+    public void givenAppealCreatedEventAndSubscriptionType_shouldInferRightSubscriptionToCreateNotification(
```

Comment:
Change this test to use the AppealLapsed notification instead since this ticket and work is about that event type, please.